packages/components/menu/src/menu.ts

```diff
@@ -15,8 +15,7 @@ import { Resize } from '@element-plus/directives'
 import Menubar from '@element-plus/utils/menu/menu-bar'
 import ElMenuCollapseTransition from './menu-collapse-transition.vue'
 import ElSubMenu from './submenu.vue'
-import useMenuColor from './useMenuColor'
-
+import { useMenuCssVar } from './useMenuCssVar'
```

Comment:
How about replacing `useMenuCssVar` with `use-menu-css-var`? (or maybe shorter?)

Reply:
Other file in this component is camelCase style. 
![image](https://user-images.githubusercontent.com/25154432/134217904-4ce22dd6-adf5-47be-b71f-b2e683630bcc.png)

And i find vueuse, `useXxx` file also use camelCase. So just keep it as it is?
> https://github.com/vueuse/vueuse

Reply:
We are unifying the naming of files and will gradually change to kebab-case in the future.

Reply:
Ok, i will change `useMenu` `useMenuColor` file too.